docs/next.config.js

```diff
@@ -147,6 +147,17 @@ module.exports = withNextPluginPreval({
         destination: '/react/:nav',
         permanent: true,
       },
+
+      /**
+       * Renamed the "headless" Authenticator docs to "advanced"
+       */
+      {
+        source:
+          '/:platform(react|react-native|angular|vue)/connected-components/authenticator/headless',
+        destination:
+          '/:platform(react|react-native|angular|vue)/connected-components/authenticator/advanced',
```

Comment:
nit: I wonder if it works without `(react|react-native|angular|vue)`

```
'/:platform/connected-components/authenticator/advanced'
```

Reply:
I'll follow up with another PR if it works